underworld/mesh/_mesh.py

```diff
@@ -486,14 +489,20 @@ def _get_iterator(self):
         iset.addAll()
         return iset._get_iterator()
 
-    def save( self, filename ):
+    def save(self, filename, units=None, time=None):
```

Comment:
@rbeucher can `time` have units, and if so, how does the user set them?

Reply:
Well time is stored as a string. It is here for convenience if you want to save attributes... We could generalize this and allow users to save any attributes...

Reply:
Okay, let's discuss further at 2pm.